### PR TITLE
Small-patch

### PR DESCRIPTION
**Pull request by jakobrosenberg**
Some small last minute fixes.

Fixes
- main.py/boot.py incorrectly shown as missing if faulty
- notification choices were stored in ./vscode/settings.json but should be stored globally

### CODE REVIEW

File: test/suite/integration/probs.config.js [hunk 1]
```diff
@@ -26,7 +27,8 @@ const prepDevice = async (device) => {
 };
 
 module.exports = {
-  async setupFile() {
+  async setupFile(file) {
+    console.log("[PREP]", file);
```
Comment (elmasria) [inline test/suite/integration/probs.config.js:31]: make sure you remove the logs

**Review by elmasria — COMMENTED** [review]
Thank you seems great, just make sure we remove the logs

File: test/suite/integration/probs.config.js (hunk 1 above)
Comment (jakobrosenberg) [reply test/suite/integration/probs.config.js:31]: Those are strictly for testing. They show when a new file is processed. :)

**Pull request merged**